public/js/helpers.js

```diff
@@ -0,0 +1,20 @@
+// should edit
+const ErrorUsernameTaken = () => {
+  let error = '<p>username taken</p>';
+  document.getElementById('message').insertAdjacentHTML('beforebegin', error);
+  return;
```

Comment:
Again, this would be a great implementation if we were opting for a client side rendering approach (single page app) for this sprint